### PR TITLE
fix(grouping): Improve copy

### PR DESCRIPTION
**Pull request by untitaker**
Docs aren't merged yet so we can't fully finalize this either.
Screenshot 2021-07-29 at 17 53 04
Screenshot 2021-07-29 at 17 53 22

### CODE REVIEW

File: static/app/views/organizationGroupDetails/grouping/errorMessage.tsx [hunk 1]
```diff
@@ -52,7 +52,9 @@ function ErrorMessage({error, groupId, onRetry, orgSlug, projSlug}: Props) {
         };
       case 'missing_feature':
         return {
-          title: t('This project does not have the grouping tree feature'),
+          title: t(
+            'This project does not have the grouping breakdown available. Is your organization still an early adopter?'
```
Comment (jjbayer) [inline static/app/views/organizationGroupDetails/grouping/errorMessage.tsx:56]: We will link to docs here once docs are live, right?

Comment (untitaker) [reply static/app/views/organizationGroupDetails/grouping/errorMessage.tsx:56]: yeah that's the idea

File: static/app/views/organizationGroupDetails/grouping/grouping.tsx [hunk 2]
```diff
@@ -45,6 +47,24 @@ type GroupingLevel = {
   isCurrent: boolean;
 };
 
+function LinkFooter() {
+  return (
+    <Footer>
+      {/* TODO(markus): Add docs btn here */}
+      <Button
+        external
```
Comment (matejminar) [inline static/app/views/organizationGroupDetails/grouping/grouping.tsx:55]: Nit: You could use `<ExternalLink />` instead.

**Review by matejminar — APPROVED** [review]
For the internal audience, this looks good to me.

Perhaps @Jesse-Box could polish this a bit before releasing it wider. (e.g. that Give Feedback button looks a bit out of place)

**Review by jjbayer — APPROVED** [review]

**Pull request merged**